### DISCUSSION TITLE
Can this be done currently? Sending custom sequences with key presses

### DISCUSSION THREAD

**Discussion by agluck91** [blinksh/blink#1886] [Q&A] [answered]
I use Alacritty on Mac, are these key bindings possible on blink? I would like to be able to make top level bindings that get sent to tmux like Alacritty does?

![image](https://github.com/blinksh/blink/assets/113385018/579b350c-416a-419c-8386-1a23f4576fdf)

**Comment by carloscabanero** [comment, marked as answer]
This is currently possible through the "Custom Presses" on input. Very similar to: https://github.com/blinksh/blink/discussions/1887

Try it out!
I have also created https://github.com/blinksh/blink/issues/1888, to make this easier.
